### PR TITLE
Add some library and python related documentation

### PR DESCRIPTION
**Pull request by hooperfly**
Added some documentation related to the PPRZLINK  `python` library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,127 @@
...
+
+Add Libraries to the Search Path
+
+    # if PAPARAZZI_SRC not set, then assume the tree containing this file is a reasonable substitute
+    PPRZ_SRC = getenv("PAPARAZZI_SRC", path.normpath(path.join(path.dirname(path.abspath(__file__)), '~/paparazzi/')))
+
+    sys.path.append(PPRZ_SRC + "/sw/lib/python")
+    sys.path.append(PPRZ_SRC + "/sw/ext/pprzlink/lib/v1.0/python")
+
+    from ivy_msg_interface  import IvyMessagesInterface
```
Comment (flixr) [inline README.md]: this is not needed...

**Comment by flixr** [comment]
Maybe link to http://docs.paparazziuav.org/latest/paparazzi_messages.html ?

File: README.md (hunk 1 above)
Comment (hooperfly) [inline README.md]: @flixr you mean the following line:
`from ivy_msg_interface import IvyMessagesInterface`

Comment (flixr) [inline README.md]: sorry, clicked on the wrong line
`sys.path.append(PPRZ_SRC + "/sw/lib/python")`
is not needed

Comment (hooperfly) [inline README.md]: Rodger that. Thanks.

**Pull request merged**